external/llvh/lib/Support/Unix/Program.inc

```diff
@@ -200,7 +200,11 @@ static bool Execute(ProcessInfo &PI, StringRef Program,
   // If this OS has posix_spawn and there is no memory limit being implied, use
   // posix_spawn.  It is more efficient than fork/exec.
 #ifdef HAVE_POSIX_SPAWN
+#if defined(__APPLE__) && defined(TARGET_OS_TV) && TARGET_OS_TV
```

Comment:
Thank you for explaining what was going on with `MemoryLimit` here. I think the right thing to do here would be to restrict the ifdef to just the region below with the `fork`, and then use `ErrMsg` to report an error if we're on tvOS and a `MemoryLimit` is specified. I suspect something similar would happen on iOS anyway, since I don't think we're allowed to actually call `fork`.